lib/WP_Auth0_LoginManager.php

```diff
@@ -560,12 +560,12 @@ private function do_login( $user, $userinfo, $is_new, $id_token, $access_token,
 	 * @link https://codex.wordpress.org/Plugin_API/Action_Reference/wp_logout
 	 */
 	public function logout() {
-		$is_sso        = (bool) $this->a0_options->get( 'sso' );
-		$is_slo        = (bool) $this->a0_options->get( 'singlelogout' );
-		$is_auto_login = (bool) $this->a0_options->get( 'auto_login' );
+		if ( ! WP_Auth0::ready() ) {
+			return;
```

Comment:
so if you call this when the SDK is not ready (I suppose) this call would return without an error? How can they know they need to wait and retry?

Reply:
The `ready` here is checking if the plugin is configured. If not, then we don't need to do anything during this process.